util/downloadBuild.py

```diff
@@ -405,8 +404,10 @@ def mozPlatform(arch):
         raise Exception("The arch passed to mozPlatform must be '64', '32', or None")
 
 
-def defaultBuildType(repoName, arch, asan, debug):
+def defaultBuildType(repoName, arch, debug, asan=None):
     """Return the default build type as per RelEng, e.g. mozilla-central-macosx-debug."""
+    if asan is None:
+        asan = False
```

Comment:
These lines aren't really necessary. `asan` is only used below and `None` evaluates false anyways. You could also make the default for the argument `False`.